persistence/jdbc/pom.xml

```diff
@@ -21,6 +21,27 @@
       </dependency>
 
       <dependency>
+         <groupId>com.zaxxer</groupId>
+         <artifactId>HikariCP</artifactId>
+         <optional>true</optional>
```

Comment:
I'm not convinced that we should use optional here. If the docs says that Hikari become our default pool, why not making it mandatory here. If we did this, we wouldn't need to modify none of the integration test poms.